app/views/my/solutions/_show_finished_independent.html.haml

```diff
@@ -0,0 +1,44 @@
+- num_avaliable_exercises = @user_track.num_avaliable_exercises
+
+.finished-section
+  .title-block
+    %h3
+      =icon("check-circle", "checkmark")
+      Well done on solving this exercise!
+  .next-block
+    %h3
+      =graphical_icon("question-circle")
+      What's next?
+
+    -if @solution.published?
+      =render "my/solutions/widgets/published"
+    -elsif @solution.completed?
+      =render "my/solutions/widgets/publish_prompt"
+    -else
+      =render "my/solutions/widgets/complete_prompt"
+
+    .next-option
+      -if @user_track.mentoring_slots_remaining?
+        %p
+          %strong Request mentor feedback.
+          Mentoring is disabled by default in Independent Mode. However, you can request feedback on a maximum of one solution at a time.
+        =link_to "Request mentor feedback", request_mentoring_my_solution_path(@solution), method: :patch, class: 'pure-button'
+      -else
+        %p
+          %strong Request mentor feedback (disabled).
+          Mentoring is disabled by default in Independent Mode. However, you can request feedback on a maximum of one solution at a time.
+        =link_to "Request mentor feedback", "#", class: 'pure-button disabled', disabled: true
+
+    =render "my/solutions/widgets/community_solutions"
+
+    -if num_avaliable_exercises.size > 0
```

Comment:
See previous: avaliable And in line 39.

Reply:
Or maybe just invent the new word :-) It occurs on more places, also not touched by this PR